fmriprep/interfaces/multiecho.py

```diff
@@ -15,8 +15,7 @@
 >>> os.chdir(datadir)
 
 """
-import os
-from nibabel.filename_parser import splitext_addext
+import os.path
```

Comment:
no need to change this?
```suggestion
import os
```